Angular/src/app/data.service.ts

```diff
@@ -1,84 +1,89 @@
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
+import { environment } from '../environments/environment'
 
 //list of all available routes:
-//api address: 104.248.116.233:5000
+//api a`dress: ${this.baseUrl}:5000
```

Comment:
Should this maybe say "address" rather than "a`dress"?

Reply:
@Jyckle Fixed! Thank you :)